src/libraries/sendtohelix.proj

```diff
@@ -41,6 +41,13 @@
     <DotNetCliVersion>$([System.Text.RegularExpressions.Regex]::Match($(GlobalJsonContent), '(%3F&lt;="dotnet": ").*(%3F=")'))</DotNetCliVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetsMobile)' == 'true'">
```

Comment:
This might break other things because it will make sure dotnet SDK is installed before the Helix job is started. I guess we don't want that?

Reply:
This is needed as we're currently installing xharness in the machine using dotnet, so we need dotnet to be installed and in the path. 

I would expect that the Helix SDK had a switch for xharness, something like:

```xml
<IncludeXharness>true</IncludeXharness>
<XharnessVersion>5.0.0-....</XharnessVersion>
```

and then helix installing xharness and dotnet for us, that way we just set the xharness switch and don't have to do the HelixPreCommands trick to install xharness.

Reply:
Currently, there are many things that have to be done as part of the XHarness job on the machines (and they differ between Android and iOS) and some of them cannot be done from within XHarness (like this installation). However, we provide exactly this functionality already in the Helix SDK.

The way it works is following:
1. You define `<XHarnessiOSProject>` items for every app bundle you want to run
2. We take it from there and zip it, slap onto it the `.sh` that prepares the env (boots the simulators/emulators and so on)
3. We prepare the Helix job `PreCommands` to install the right runtime and the tool itself
4. We make sure the logs are written in the right place and xUnit results are uploaded back to AzDO

This whole functionality is now being duplicated in this PR and would require way more work for it to work because there are many caveats around (such as running in a GUI-rendering supporting user session and so on..)

The XHarness CLI version is defined in `Versions.props` in this repository and can also be overriden via props.